jjb/install-rhel7.yaml

```diff
@@ -31,9 +31,16 @@
     builders:
       - shell: |
             #!/bin/bash -ex
+            virtualenv ansible.venv && source ansible.venv/bin/activate
+            pip install --upgrade setuptools
+            pip install 'ansible>=2.1'
```

Comment:
If this was tested successfully on Ansible 2.1, let's set this to lock the version with '==', since new versions of Ansible can break existing playbooks.

Reply:
I think ansible 2.4.1.0 was installed on the master and I can pin the version to that but how do I know if everything is working as expected ? Ping me on IRC and I can provide you with internal link to this job.

Reply:
Note2: `jjb/code-coverage.yaml` also uses 'ansible>=2.1' syntax.

Reply:
I've done some testing today, and my current installer playbook is compatible with the latest release of Ansible, so please update this PR with the following Ansible version specification:

ansible==2.4.1.0